scrapers/SARJ-LLC.yml

```diff
@@ -0,0 +1,42 @@
+name: "SARJ LLC"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metart.com
+      - sexart.com
+      - thelifeerotic.com
+      - errotica-archives.com
+      - alsscan.com
+      - vivthomas.com
+      - metartx.com
+      - rylskyart.com
+      - eternaldesire.com
+      - stunning18.com
+      - lovehairy.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="info-container"]
+      $script: //script[contains(text(),'INITIAL_STATE')]/text()
+    scene:
+      Title: $info//h3/a/text()
+      Details:
+        selector: $info//p[@class="description-text"]/text()
```

Comment:
in `https://www.sexart.com/model/nancy-a-and-eric-el-gran-and-melody-petite-and-ariel-piper-fawn-and-nick-ross-and-alexis-crystal/movie/20200621/SEXART_HOLIDAY_ON_MYKONOS_EPISODE_6` or `https://www.sexart.com/model/sylvan-and-julia-roca-and-carolina-abril/movie/20200506/THREE_IN_A_BED` or `https://www.thelifeerotic.com/model/lil-karla/movie/20200618/FORCED_ORGASM_2` you can see that you get the short description not the normal.
The normal one at least for sexart, tle is in the script you have in common i think (that you dont seem to use anywhere)

Reply:
Updated to use the script instead, tested and it seems to work on all the sites that have descriptions to scrape.